dbms/tests/integration/test_relative_filepath/test.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance('node')
+path_to_userfiles_from_defaut_config = "user_files"
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+def test_filepath(start_cluster):
+    # 2 rows data
+    some_data = "\t111.222\nData\t333.444"
+
+    node.exec_in_container(['bash', '-c', 'echo "{}{}" > {}'.format(
+        filename,
```

Comment:
Looks like this test doesn't work:
`NameError: global name 'filename' is not defined`

Please, merge `master` into your branch before fixing it (now you can see `Integration tests — failed:0, passed:0, error:0`, it's because integration tests was broken in `master`)